guide/java/policies.md

```diff
@@ -31,42 +31,144 @@ Policies are highly reusable as their inputs, thresholds and targets are customi
 
-   e.g. if the average request per second across a cluster of Tomcat servers goes over the high watermark, it will resize the cluster to bring the average back to within the watermarks.
-  
-<!---
-TODO - list some
-TODO - describe how they can be customised (briefly mention sensors)
--->
+e.g. if the average request per second across a cluster of Tomcat servers goes over the high watermark, it will resize the cluster to bring the average back to within the watermarks.
+
+{% highlight yaml %}
+
+  - type: org.apache.brooklyn.policy.autoscaling.AutoScalerPolicy
+    brooklyn.config:
+      metric: webapp.reqs.perSec.perNode
+      metricUpperBound: 3
+      metricLowerBound: 1
+      resizeUpStabilizationDelay: 2s
+      resizeDownStabilizationDelay: 1m
+      maxPoolSize: 3
+
+{% endhighlight %}
+
+#### ServiceReplacer
+
+Attaches to a DynamicCluster and replaces a failed member in response to HASensors.ENTITY_FAILED or other sensor.  The [introduction to policies](../) shows a worked example of the ServiceReplacer policy in user.
+
+#### FollowTheSunPolicy
+
+Policy for moving work around to follow the demand.  The work can be any Movable entity.  This currently available in yaml blueprints.
+
+#### ConditionalSuspendPolicy
+
+This policy will suspend and resume a target policy based on configured suspend and resume sensors.
+
+#### CreateUserPolicy
+
+Attaches to an Entity and monitors for the addition of a location to that entity, the policy then adds a new user to the VM with a randomly generated password, with the SSH connection details set on the entity as the createuser.vm.user.credentials sensor.
+
+#### AdvertiseWinRMLoginPolicy
+
+This is similar to the CreateUserPolicy.  It will monitor the addition of WinRmMachineLocations to an entity and then create a sensor advertising the administrative user's credentials.
+
+#### SshMachineFailureDetector
+
+HA policy for monitoring an SshMachine, emitting an event if the connection is lost/restored.
+
+#### ConnectionFailureDetector
+
+HA policy for monitoring an http connection, emitting an event if the connection is lost/restored.
 
+#### StopAfterDurationPolicy
+
+The StopAfterDurationPolicy can be used to limit the lifetime of an entity.  After a configure time period expires the entity will be stopped.
+
+#### LoadBalancingPolicy
+
+Policy that is attached to a pool of "containers", each of which can host one or more migratable "items".  The policy monitors the workrates of the items and effects migrations in an attempt to ensure that the containers are all sufficiently utilized without any of them being overloaded.
+
+#### ServiceRestarter
+
+Attaches to a SoftwareProcess (or anything Startable, emitting ENTITY_FAILED or other configurable sensor), and invokes restart on failure; if there is a subsequent failure within a configurable time interval, or if the restart fails, this gives up and emits {@link #ENTITY_RESTART_FAILED}
```

Comment:
Let's re-order, based on what we expect people to use most often. The first few should be:
AutoScaler, ServiceRestarter, StopAfterDurationPolicy, CreateUserPolicy, AdvertiseWinRmLoginPolicy, SshMachineFailureDetector, ConnectionFailureDetector, and then all the rest.